lib/private/AppFramework/DependencyInjection/DIContainer.php

```diff
@@ -315,6 +315,17 @@ public function __construct(string $appName, array $urlParams = [], ServerContai
 				$c->get(\OC\AppFramework\Middleware\AdditionalScriptsMiddleware::class)
 			);
 
+			/** @var OC\AppFramework\Bootstrap\Coordinator $coordinator */
+			$coordinator = $c->get(OC\AppFramework\Bootstrap\Coordinator::class);
```

Comment:
```suggestion
			/** @var \OC\AppFramework\Bootstrap\Coordinator $coordinator */
			$coordinator = $c->get(\OC\AppFramework\Bootstrap\Coordinator::class);
```

Or full import please